src/common/utils.go

```diff
@@ -308,3 +307,36 @@ func GetNextMajorVersion(version string) (string, error) {
 
 	return strconv.Itoa(major + 1), nil
 }
+
+func GetTarantoolRepoVersion(version string) string {
```

Comment:
Does it consider that 2.1 is 2_x? However check it please, may be it has been already fixed.

Reply:
Fixed to replace `2_1` to `2x`.